src/payment/checkout/payment-form/flex-microform/FlexMicroform.jsx

```diff
@@ -38,16 +38,25 @@ class FlexMicroform extends React.Component {
       });
       return;
     }
-    window.microform = new window.Flex(captureKeyId).microform({
-      styles: {
-        input: {
-          'font-size': '16px',
-          'font-family': 'Roboto, "Helvetica Neue", Arial, sans-serif',
-          color: '#2d323e',
+    try {
+      window.microform = new window.Flex(captureKeyId).microform({
+        styles: {
+          input: {
+            'font-size': '16px',
+            'font-family': 'Roboto, "Helvetica Neue", Arial, sans-serif',
+            color: '#2d323e',
+          },
         },
-      },
-    });
-    this.props.dispatch(microformStatus(STATUS_READY));
+      });
+      this.props.dispatch(microformStatus(STATUS_READY));
+    } catch (err) {
+      if (err.reason === 'CAPTURE_CONTEXT_EXPIRED') {
+        this.props.dispatch(captureKeyProcessing(false));
+        this.props.dispatch(fetchCaptureKey());
```

Comment:
We probably want to log the issue to segment so we can track this (look for `sendTrackEvent`), we also maybe need to do something so if we somehow keep using a bad key we don't just get stuck in a loop